source/controllers/include/controllers/impedance/Dissipative.hpp

```diff
@@ -157,7 +157,7 @@ template<class S>
 Dissipative<S>::Dissipative(const ComputationalSpaceType& computational_space, unsigned int nb_dimensions) :
     Impedance<S>(Eigen::MatrixXd::Zero(nb_dimensions, nb_dimensions),
                  Eigen::MatrixXd::Identity(nb_dimensions, nb_dimensions),
-                 Eigen::MatrixXd::Identity(nb_dimensions, nb_dimensions)),
+                 Eigen::MatrixXd::Zero(nb_dimensions, nb_dimensions)),
```

Comment:
I guess this is a design choice. If the input state contains acceleration, then Identity treats it as a direct feedforward. Zero on the other hand ignores it completely. So the user is explicitly responsible for setting the inertia matrix if they use acceleration inputs (and of course they can set it to Identity if they choose). Since this Dissipative wrapper doesn't deal with acceleration, I agree that Zero default makes more sense.